### PR TITLE
Add Bobcat RK3566 support

### PR DESCRIPTION
**Pull request by ccrisan**
Add support for Bobcat RK3566 while bringing this repo on par with https://github.com/NebraLtd/helium-bobcat-px30.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"applicationId":160,"deviceType":"isg-503","userId":2,"appUpdatePollInterval":600000,"listenPort":48484,"vpnPort":443,"apiEndpoint":"https://api.cloud.nebra.com","vpnEndpoint":"vpn.cloud.nebra.com","registryEndpoint":"registry.cloud.nebra.com","deltaEndpoint":"https://delta.cloud.nebra.com","mixpanelToken":"__unused__","apiKey":"kpvyuEPPxOiJHHUIOBZrCJkzkp941aY7"}
```
Comment (shawaj) [inline config.json]: Don't think this is correct as wrong device type? Did you generate from the Balena cli?

Comment (ccrisan) [reply config.json]: Nope, this is a copy from the `bobcat-px30`. I suggest that we stick with the same device type as it doesn't really matter as long as it's identified as a known one. Is there anything else that needs to be changed?

Comment (shawaj) [reply config.json]: I think the device will have to be different ... See slack message

Comment (shawaj) [reply config.json]: https://nebraltd.slack.com/archives/C024BNQ1Y6T/p1691097130419659

**Review by shawaj — APPROVED** [review]

**Pull request merged**